cmd/background-newdisks-heal-ops.go

```diff
@@ -87,6 +87,8 @@ type healingTracker struct {
 	// ID of the current healing operation
 	HealID string
 
+	ItemsSkipped uint64
+	BytesSkipped uint64
```

Comment:
Not needed in `toHealingDisk` as per comment below?

Reply:
yes, thanks. Fixed!